docs/source/tutorials/filtsmooth/particle_filtering.ipynb

```diff
@@ -2780,7 +2780,7 @@
     "    locations=locs,\n",
     ")\n",
     "\n",
-    "ode_posterior = ode_pf.filter(regression_problem)"
+    "ode_posterior, _ = ode_pf.filter(regression_problem)"
```

Comment:
I cant believe it, a readable notebook diff??

Reply:
I edited the raw file 😄